src/sbb.py

```diff
@@ -86,8 +107,10 @@ def post_ballots_and_commitments(self) -> None:
         self.post_end_section()
 
         self._db.write(ORIGINAL_ORDER_COMMITMENTS + '\n')
-        for com in self._svr_commitments:
-            self._db.write(com + '\n')
+        # TODO: fix this
```

Comment:
Hmm what needs to be fixed?

Looks like you switched from writing three lines for each SVR to one list of length 3 for each 3-element SVR?

Reply:
Just need to remove the comment. Working with mashaling/unmarshaling was kind of annoying so I kept things in memory while I worked on it